docs/photoniq/vwrs/get-started-vwrs/get-started-vwrs-native.md

```diff
@@ -0,0 +1,92 @@
+
+## Before You Begin
+
+Ensure you have completed the following prerequisites before starting the integration process:
+
+- **Coordinate with Macrometa Personnel:** Secure the necessary credentials and access rights by engaging with Macrometa personnel.
+- **Plan Your Waiting Room:** Identify the appropriate [queue type](../queue-types.md) for your application, determine which endpoints will incorporate the waiting room, and decide on the priority handling mechanisms.
+- **Collect Information:** Gather essential information, including API endpoints and expected traffic metrics, to facilitate a smooth setup.
+
+## Integration Overview
+
+Integrate VWRs with your native applications by following these steps:
+
+1. **API Key Creation:** Start by creating an API key to access the VWRs system, as this key will be essential for all API interactions. For instructions on creating an API key, refer to [Create an API Key](https://www.macrometa.com/docs/apiVwrs#/operations/createAPIKey).
+
+2. **Create and Configure a Domain:** Create a domain that hosts your waiting room. This step allows you to fine-tune how the waiting room operates and integrates with your site. For comprehensive guidance on domain configuration, refer to [Configure a Domain](../configure-domain.md).
+
+3. **API Usage for Queue Management:** Use the VWRs API to manage the user's position in the waiting room. This involves making periodic API calls to check the queue status and manage the user experience accordingly.
+
+### API Request
+
+Non-web applications use the **GET /api/vwr/v1/position/{queue_name}/{visitor_id}** REST API to check resource access.
+
+Key parameters include:
+
+- **queue_name**: Specifies the waiting room created in the VWR.
+- **visitor_id**: The _visitor_id_ parameter is crucial in tracking the user's position and wait duration. It is not provided initially but returned with each subsequent API call, ensuring accurate tracking.
+
+### API Response
+
+This API returns the following JSON structure:
+
+```json
+{
+  "position": 0,
+  "waiting_time": 0,
+  "queue_depth": 0,
+  "visitor_id": 0,
+  "refresh_interval": 0
+}
+```
+
+The response includes:
+
+- **position**: User's current position in the waiting room.
+- **waiting_time**: Estimated time (seconds) until the user exits the waiting room.
+- **queue_depth**: Total number of users in the waiting room.
+- **visitor_id**: ID for the current request.
+- **refresh_interval**: Time (in seconds) between successive position checks.
+
+When the position is zero, the user has exited the waiting room and may proceed.
+
+## Pseudo-Code Example
+
+Here is a pseudo-code example illustrating how a native app interacts with the VWR API to manage a user's waiting experience:
```

Comment:
Again "manage" is not the right word.  It implies a little that this API can influence the queue position, which it cant.  I can only tell you where you are in the queue and how long you have before they exit the queue.